src/Test/Utilities/Portable/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -182,9 +182,9 @@ public sealed override void Initialize(AnalysisContext context)
                                                      IOperation advanceIncrement = null;
                                                      BinaryOperationKind advanceOperationCode = BinaryOperationKind.None;
 
-                                                     if (advanceExpression.Kind == OperationKind.AssignmentExpression)
+                                                     if (advanceExpression.Kind == OperationKind.SimpleAssignmentExpression)
```

Comment:
Not related to this PR, but this method is getting quite deep. Consider refactoring.